### PR TITLE
Network selector UI

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/tokenIcon/TokenIconsSmall.ts [hunk 1]
```diff
@@ -85,6 +85,8 @@ const tokenIconsSmall: { [key in Ticker]?: string } = {
...
   <path d="M12.6018 15.1779C12.0898 15.9559 11.1038 16.6899 9.40582 16.5659C8.54982 16.5079 6.48982 15.4059 6.87782 12.9719C6.94582 12.5159 7.19982 11.9139 7.19982 11.9139L6.04582 12.1379L5.81982 11.2279L8.63782 10.5599L9.18582 13.4799L8.34782 13.6459L8.05582 12.2599C8.05582 12.2599 7.26782 13.4779 8.23982 14.7899C8.81582 15.4659 9.09582 15.4999 9.76182 15.6219C10.3438 15.7219 11.7118 15.5579 12.1598 13.9799C12.3578 12.9619 12.1838 12.1979 10.8918 11.1699L10.9958 10.0679C10.9958 10.0679 12.4418 11.1439 12.8018 11.8819C13.0778 12.4679 13.5518 13.6199 12.6018 15.1779V15.1779Z" fill="white"/>`,
+  FANTOM: `<path d="M12 24C18.6274 24 24 18.6274 24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 18.6274 5.37258 24 12 24Z" fill="#13B5EC"/>
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.9 9.675L15.6 8.1V11.25L12.9 9.675ZM15.6 16.425L12 18.525L8.4 16.425V12.75L12 14.85L15.6 12.75V16.425ZM8.4 8.1L11.1 9.675L8.4 11.25V8.1ZM12.45 10.425L15.15 12L12.45 13.575V10.425ZM11.55 13.575L8.85 12L11.55 10.425V13.575ZM15.15 7.35L12 9.15L8.85 7.35L12 5.475L15.15 7.35ZM7.5 7.05V16.875L12 19.425L16.5 16.875V7.05L12 4.5L7.5 7.05Z" fill="white"/>`,
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/tokenIcon/TokenIconsSmall.ts:89]: should be `fillRule`, `clipRule`

File: packages/tempus-client_v2/src/components/navbar/NavBar.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { FC, useCallback } from 'react';
 import { useNavigate } from 'react-router-dom';
 import TempusLogo from './tempusLogo';
+import ChainSelector from './ChainSelector';
 import Links from './Links';
 import Wallet from '../wallet/Wallet';
 
 import './NavBar.scss';
+import Spacer from '../spacer/spacer';
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/navbar/NavBar.tsx:9]: import should be after line 2

**Review by danieledamiani — APPROVED** [review]
a couple of minor comments

**Review by ShawTim — APPROVED** [review]

File: packages/tempus-client_v2/src/components/navbar/NavBar.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/navbar/NavBar.tsx]: fixed

File: packages/tempus-client_v2/src/components/tokenIcon/TokenIconsSmall.ts (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/tokenIcon/TokenIconsSmall.ts]: fixed

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**